src/Collector/Formatter.php

```diff
@@ -23,20 +22,8 @@
  */
 class Formatter implements MessageFormatter
 {
-    /**
-     * @var MessageFormatter
-     */
-    private $formatter;
-
-    /**
-     * @var CurlCommandFormatter
-     */
-    private $curlFormatter;
-
-    public function __construct(MessageFormatter $formatter, MessageFormatter $curlFormatter)
+    public function __construct(private readonly MessageFormatter $formatter, private readonly MessageFormatter $curlFormatter)
```

Comment:
with multiple promoted constructor arguments, i find it much more readable to have each property on its own line:

```suggestion
    public function __construct(
        private readonly MessageFormatter $formatter, 
        private readonly MessageFormatter $curlFormatter,
    )
```

Reply:
++